app/components/Header.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import { faPlus, faDownload } from '@fortawesome/free-solid-svg-icons'
+
+import { Modal, ModalType } from '../models/modals'
+
+import { connectComponentToProps } from '../utils/connectComponentToProps'
+
+import { setModal } from '../actions/ui'
+
+import HeaderColumnButton from './chrome/HeaderColumnButton'
```

Comment:
Ditto on the import reorg:

```suggestion
import React from 'react'
import { faPlus, faDownload } from '@fortawesome/free-solid-svg-icons'

import { Modal, ModalType } from '../models/modals'
import { connectComponentToProps } from '../utils/connectComponentToProps'
import { setModal } from '../actions/ui'

import HeaderColumnButton from './chrome/HeaderColumnButton'
```

Having this pattern enforced makes it easier to browse the requirements of a given file